packages/shared/warningWithoutStack.js

```diff
@@ -25,16 +25,17 @@ if (__DEV__) {
     if (condition) {
       return;
     }
+
+    let argIndex = 0;
+    const message = 'Warning: ' + format.replace(/%s/g, () => args[argIndex++]);
```

Comment:
@gaearon Ended up just moving this line up a bit so the string replacement is consistent between the two blocks. Feel okay?

Reply:
Sorry, that was an intentional change. See my collapsed comment (silly GitHub)

Reply:
Okay. No worries :)